shoes-swt/lib/shoes/swt/redrawing_aspect.rb

```diff
@@ -99,6 +100,14 @@ def redraw_area(left, top, width, height, include_children = true)
         end
       end
 
+      def redraw_target(element)
+        if element.gui
+          element.gui.redraw_target
+        else
+          element
+        end
+      end
```

Comment:
So the actual reason is that we occasionally trigger the redrawing during initialization of some elements (Image in particular), so we'll get here before the `gui` is actually set in a class. Haven't dug too deeply on why we get that redraw at this point, since this seemed relatively benign.